### PR TITLE
Have made the course search bar.

### PR DESCRIPTION
**Pull request by komalahire**
**I have made a course search box in all course components. 
please have a look.**

### CODE REVIEW

File: src/components/Course/styles.scss [hunk 1]
```diff
@@ -1,10 +1,55 @@
...
+  border-radius: 8px 0 0 8px;
+  font-size: 20px;
+  outline: none;
+  padding-left: 15px;
+  @include breakpoint-mobile {
+    font-size: 17px;
+  }
+}
+
+.searchTerm:focus {
```
Comment (vikotar) [inline src/components/Course/styles.scss]: It should be `search-term` I guess.

File: src/components/Course/styles.scss [hunk 2]
```diff
@@ -1,10 +1,55 @@
...
+  position: relative;
+  display: flex;
+  margin: auto;
+  @include breakpoint-mobile {
+    width: 95%;
+    margin: 0 auto;
+  }
+}
+
+.search-term {
```
Comment (vikotar) [inline src/components/Course/styles.scss:26]: Nice work with css.  👍

**Comment by vikotar** [comment]
@komalahire  could we put the color `#83B344` green instead of red in the search bar? I think it looks better. 
cc @pysaquib  and btw Nice work with the search bar.
Screenshot 2020-12-20 at 2 54 41 AM

File: src/components/Class/ClassList/styles.scss [hunk 3]
```diff
@@ -32,6 +32,11 @@ th {
   font-family: "Times New Roman", Times, serif;
 }
 
+
+.msg{
```
Comment (vikotar) [inline src/components/Class/ClassList/styles.scss]: space after message. `.msg {`

File: src/components/Course/index.js [hunk 4]
```diff
@@ -19,16 +20,51 @@ function Course() {
...
-      <CourseList
-        list={get(data, "enrolledCourses")}
-        title="Aap in courses mein enrolled hai"
-      />
-      <CourseList
-        list={get(data, "allCourses")}
-        title="Aap yeh courses mein enroll kar skte hai"
-      />
+    <div>
+      <div className="search">
```
Comment (vikotar) [inline src/components/Course/index.js:37]: could we create a task to move this search bar to its own independent component later?

File: src/components/Course/index.js [hunk 5]
```diff
@@ -19,16 +20,51 @@ function Course() {
...
+        <input
+          type="text"
+          className="search-term"
+          placeholder="What are you looking for?"
```
Comment (vikotar) [inline src/components/Course/index.js]: Perhaps the better placeholder could be.. `Search for a course. ex. Python`
What do you think? cc @pysaquib

File: src/constant.js [hunk 6]
```diff
@@ -6,5 +6,6 @@ export const PATHS = {
   CLASS: "/class",
   COURSE: "/course",
   COURSE_CONTENT: "/course/:courseId",
+  COURSE_SEARCH:"/COURSE_SEARCH"
```
Comment (vikotar) [inline src/constant.js]: We might not need this.

**Review by vikotar — COMMENTED** [review]
Nice work, @komalahire. I have put some comments. Let me know if I wasn't clear enough.

File: src/components/Course/styles.scss (hunk 1 above)
Comment (komalahire) [reply src/components/Course/styles.scss]: Done

File: src/components/Course/index.js (hunk 4 above)
Comment (pysaquib) [reply src/components/Course/index.js:37]: @komalahire Please take note and add this in the issues

File: src/components/Course/index.js (hunk 5 above)
Comment (pysaquib) [reply src/components/Course/index.js]: Makes sense

Comment (komalahire) [reply src/components/Course/index.js]: done

File: src/constant.js (hunk 6 above)
Comment (komalahire) [reply src/constant.js]: yes. no need, I removed

File: src/components/Course/index.js (hunk 4 above)
Comment (komalahire) [reply src/components/Course/index.js:37]: yes. sure

File: src/pages/CourseContent/index.js [hunk 7]
```diff
@@ -76,6 +82,12 @@ function CourseContent(props) {
   return (
     <div className="ng-course-content">
       <div className="content">
+        <h1>{courseTitle}</h1>
+        <h2>{get(selectedExercise,"exercise.name")}</h2>
+        <ExerciseContent content={get(selectedExercise, "exercise.content")} />
+        <EditOnGithub
+          link={`${get(selectedExercise, "exercise.githubLink")}`}
+        />
```
Comment (vikotar) [inline src/pages/CourseContent/index.js]: Try to remove that. Probably we don't need these. The below code takes care of it.

File: src/pages/Login/index.js [hunk 8]
```diff
@@ -22,18 +22,22 @@ function Login(props) {
...
     };
+  
     // let's send the data to our backend.
     dispatch(userActions.onUserSignin(googleData));
+   
+
   }
 
   const onGoogleLoginFail = (errorResponse) => {
+    console.log("komabhatt")
```
Comment (vikotar) [inline src/pages/Login/index.js]: Could we remove this console.log? 🙂

**Review by vikotar — APPROVED** [review]
Nice work, @komalahire 👍

**Pull request merged**